src/patterns/configurable-columns-pattern/__spec__.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ImmutableHelper from './../../utils/helpers/immutable';
+import ConfigurableColumnsPattern from './configurable-columns-pattern';
+import { ConfigurableColumns, ConfigurableColumnRow } from './../../components/configurable-columns';
+
+describe('ConfigurableColumnsPattern', () => {
+  let wrapper
+  let data = ImmutableHelper.parseJSON(
+    [
+      { id: 1, name: 'Foo', locked: true, enabled: true },
+      { id: 2, name: 'Bar', locked: false, enabled: true },
+      { id: 3, name: 'Baz', locked: false, enabled: false }
+    ]
+  );
+  let onCancel = () => { }
+  let onChangeSpy = jasmine.createSpy('onChangeSpy')
+  let onChange = (id) => { onChangeSpy(id) }
+  let onClick = () => { }
```

Comment:
Again, I'd like to check these empty functions aren't all identical so that `expect(onClick).toEqual(onCancel)` fails